tests/src/core/testqgsstyle.cpp

```diff
@@ -555,26 +555,26 @@ void TestStyle::testLoadColorRamps()
 
   // values for color tests
   QMultiMap< QString, QPair< double, QColor> > colorTests;
-  colorTests.insert( QStringLiteral( "test_gradient" ), qMakePair( 0.25, QColor( "#ff8080" ) ) );
-  colorTests.insert( QStringLiteral( "test_gradient" ), qMakePair( 0.66, QColor( "#aeaeff" ) ) );
+  colorTests.insert( QStringLiteral( "test_gradient" ), qMakePair( 0, QColor( "#ff0000" ) ) );
+  colorTests.insert( QStringLiteral( "test_gradient" ), qMakePair( 1, QColor( "#0000ff" ) ) );
```

Comment:
@nyalldawson , should we keep at least one intermediary step, but put it at 0.5 instead (which I assume would be stable)?

Reply:
There's (better) tests for this elsewhere -- here we are just testing that ramps actually get restored ( just checking the names alone would probably work for that!)

Reply:
@nyalldawson , roger that, approved.